src/util/string.ts

```diff
@@ -16,4 +16,5 @@ export function encodeHtml(html: string) {
     .replace(/>/g, '&gt;')
     .replace(/'/g, '&#39;')
     .replace(/"/g, '&#34;')
+    .replace(/\./g, '&middot;') // replaces period with middle dot, to ensure that the title is not misinterpreted as a URL
```

Comment:
I was finding that various tools were getting confused by link titles that contained special characters (both in Markdown and HTML). This was a simple solution that solved it, as all the regexes of the various tools seem to require at least one `.` to consider something a URL.